tests/unit/DocumentTest.php

```diff
@@ -112,6 +112,15 @@ public function testGetTimeExceptionOnNonUuid()
         $this->expectExceptionCode(DatabaseException::ERR_INVALID_UUID);
         $this->document->getTime();
     }
+    
+    public function testGetTimeExceptionOnNonUuidV1()
+    {
+        $this->expectException(DatabaseException::class);
+        $this->expectExceptionCode(DatabaseException::ERR_INVALID_UUID);
+
+        $this->document->setId('e30ee5de-f7a0-41f0-837c-d3cfec4c7cc7');
```

Comment:
I'm not sure this case adds any test value beyond the existing test above for the same exception on a non-UUIDv1 document ID. It doesn't matter whether the ID string is `12345`, a v4 UUID, `foobar` or anything else in terms of triggering the exception on `getTime()`.

Reply:
https://github.com/dwgebler/doclite/blob/main/src/Document.php#L382-L384

```php
if (!$uuid instanceof UuidV1) {
    throw new InvalidArgumentException('Invalid UUID, not v1');
}
```

The test covers this condition, existing tests do not go there

Reply:
This condition is tested in https://github.com/dwgebler/doclite/blob/main/tests/unit/DocumentTest.php#L109-L114